R/utils.R

```diff
@@ -9,6 +9,24 @@ phantom_run <- function(args, wait = TRUE) {
   system2(phantom_bin, args = args, wait = wait)
 }
 
+
+# Try really hard to find bower in Windows
+find_phantom <- function(){
+  # a slightly more robust finder of bower for windows
+  # which does not require PATH environment variable to be set
+  phantom_path = if(Sys.which("phantomjs") == "") {
+    # if it does not find Sys.which('bower')
+    # also check APPDATA to see if found there
+    if(identical(.Platform$OS.type,"windows")) {
+      Sys.which(file.path(Sys.getenv("APPDATA"),"npm","phantomjs."))
+    }
+  } else {
+    Sys.which("phantomjs")
+  }
+  return(phantom_path)
```

Comment:
Thanks. :) With that convention, the `return()` serves as an indicator that something unusual is happening.